etc/scripts/parsed-pug/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "type": "commonjs"
+}
```

Comment:
This webpack loader has to be defined as commonjs, otherwise it won't work with Webpack's dynamic importing